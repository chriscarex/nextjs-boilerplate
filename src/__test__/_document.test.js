import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MyDocument from '../pages/_document';

const setup = () => {
  const props = {};

  const component = shallow(<MyDocument {...props} />);

  return {
    component,
    props
  };
};

describe('_document', () => {
  it('should match snapshot ', () => {
    const {
      component
    } = setup();

    expect(toJson(component)).toMatchSnapshot();
  });
});
