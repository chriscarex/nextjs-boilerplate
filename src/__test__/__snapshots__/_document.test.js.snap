// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_document should match snapshot  1`] = `
<Html
  lang="en"
>
  <Head>
    <meta
      content="IE=Edge"
      httpEquiv="x-ua-compatible"
    />
    <meta
      content="block-all-mixed-content;default-src 'none';connect-src 'self';font-src fonts.googleapis.com;form-action 'self';img-src 'self' data:;manifest-src 'self';prefetch-src 'self';script-src 'self';style-src 'self';"
      httpEquiv="Content-Security-Policy"
    />
    <meta
      content="nosniff"
      httpEquiv="X-Content-Type-Options"
    />
    <meta
      content="max-age=31536000"
      httpEquiv="Strict-Transport-Security"
    />
    <meta
      content="strict-origin-when-cross-origin"
      httpEquiv="Referrer-Policy"
    />
    <meta
      lang="en"
    />
    <link
      href="/img/bee.svg"
      rel="shortcut icon"
    />
  </Head>
  <body>
    <Main />
    <NextScript />
  </body>
</Html>
`;
